cdc/sinkv2/tablesink/event_table_sink.go

```diff
@@ -58,18 +74,22 @@ func (e *eventTableSink[E]) UpdateResolvedTs(resolvedTs model.ResolvedTs) {
 		return
 	}
 	resolvedEvents := e.eventBuffer[:i]
+	e.eventBuffer = append(make([]E, 0, len(e.eventBuffer[i:])), e.eventBuffer[i:]...)
```

Comment:
How about add a comment to explain why do you to reset the `eventBuffer` in this way.